src/main/java/core/basesyntax/random/FigureSupplier.java

```diff
@@ -0,0 +1,74 @@
+package core.basesyntax.random;
+
+import core.basesyntax.figure.Figure;
+import core.basesyntax.figuretype.Circle;
+import core.basesyntax.figuretype.IsoscelesTrapezoid;
+import core.basesyntax.figuretype.Rectangle;
+import core.basesyntax.figuretype.RightTriangle;
+import core.basesyntax.figuretype.Square;
+import java.util.Random;
+
+public class FigureSupplier {
+    public static final int MAX_VALUE = 10;
+    private Random random = new Random();
+    private ColorSupplier randomColor = new ColorSupplier();
+    private String color = randomColor.getRandomColor();
+
+    public Figure getRandomFigure() {
+        switch (random.nextInt(MAX_VALUE)) {
+            case 0:
+                return circle();
+            case 1:
+                return isoscelesTrapezoid();
+            case 2:
+                return rectangle();
+            case 3:
+                return rightTriangle();
+            default:
+                return square();
+        }
+    }
+
+    public int getRandomValue(int maxValue) {
+        return random.nextInt(maxValue);
+    }
+
+    private Circle circle() {
```

Comment:
```suggestion
    private Circle getCircle() {
```

названия методов начинаем с глаголов. Поменяй во всех следующих методах тоже